src/main/scala/org/apache/mesos/chronos/scheduler/jobs/JobStats.scala

```diff
@@ -310,4 +447,15 @@ class JobStats @Inject()(clusterBuilder: Option[Cluster.Builder], config: Cassan
     }
 
   }
+
+  private def readConsistencyLevel() : ConsistencyLevel = {
+    var level = ConsistencyLevel.ANY
```

Comment:
You can omit this `level` var altogether. The last statement in the function will be the return value.